modular_skyrat/modules/modular_implants/code/nifsofts/dorms.dm

```diff
@@ -58,3 +58,49 @@
 					/obj/item/clothing/shoes/latex_heels,
 	)
 	purchase_price = 200
+
+/obj/item/disk/nifsoft_uploader/dorms/contract
+	name = "\improper Purpura Contract"
```

Comment:
The disk needs a desc
It needs to be clear on examine what the disk uploads, because the name isn't obvious

Reply:
There is an automatic description that shows what the NIFSoft does on examine.